### PR TITLE
example of page fragments usage

### PR DESCRIPTION
**Pull request by dmitriyfolio**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {
-  "baseUrl": "https://folio-snapshot.dev.folio.org",
+  "baseUrl": "http://localhost:3000/",
```
Comment (KyryloBrener) [inline cypress.json:2]: Do we need these changes here?

Comment (dmitriyfolio) [reply cypress.json:2]: no. And may be it's good to add config file example into md file and add such files into .gitignore. Need to discuss with the all Team.

File: cypress/plugins/dateTools.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+export function getCurrentDate(){
```
Comment (NikitaSedyx) [inline cypress/plugins/dateTools.js:2]: it's not plugin, just util

**Comment by NikitaSedyx** [comment]
plz fix linter issues + EOF

File: cypress.json (hunk 1 above)
Comment (dmitriyfolio) [reply cypress.json:2]: I will revert this change at the end of review, before merge in master

File: cypress/plugins/dateTools.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/plugins/dateTools.js:2]: we don't have such folder. Do you mean I should add new folder utils in the root?

File: cypress.json (hunk 1 above)
Comment (dmitriyfolio) [reply cypress.json]: done

File: cypress/plugins/dateTools.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/plugins/dateTools.js]: moved to utils in support

**Review by KyryloBrener — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**